recipe/meta.yaml

```diff
@@ -29,13 +29,16 @@ requirements:
     - m2-patch            # [win]
   host:
```

Comment:
The host should be:
```
  host:
    - python
    - cython >=0.29.22
    - freetds
    - pip
    - setuptools >=54.0
    - setuptools-git
    - setuptools_scm >=5.0
    - toml
    - wheel >=0.36.2
```
see https://github.com/pymssql/pymssql/blob/v2.2.3/pyproject.toml